components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/util/ConfigurationUtils.java

```diff
@@ -130,12 +147,54 @@ public static boolean useCreatedTimeField() {
 
     private static String populateMessageWithData(ConfigurationConstants.ErrorMessages error, String data) {
 
-        String message;
-        if (StringUtils.isNotBlank(data)) {
-            message = String.format(error.getMessage(), data);
-        } else {
-            message = error.getMessage();
+        return String.format(error.getMessage(), data);
+    }
+
+    private static String populateMessageWithData(ConfigurationConstants.ErrorMessages error) {
+
+        return error.getMessage();
+    }
+
+    /**
+     * Creates the file endpoint using the parameters.
+     *
+     * @param fileId file id.
+     * @param resourceName resource name.
+     * @param resourceType resource type name.
+     * @return tenant domain of the request is being served.
+     */
+    public static String getFilePath(String fileId, String resourceType, String resourceName) {
+
+        return RESOURCE_PATH + '/' + resourceType + '/' + resourceName + '/' + FILE + '/' + fileId;
```

Comment:
Shall we use constant for '/' ?

Reply:
FIxed https://github.com/wso2/carbon-identity-framework/pull/2690